### ISSUE TITLE
Per-space and global page templates

### ISSUE THREAD

**Issue by pztrn** [docmost/docmost#175]
It will be super useful to have page templates, e.g. when you have to make one page per some incident which should be filled exactly the same.

**Comment by Shadowfita** [comment]
Yes please! This would be great. Currently this can be achieved by exporting a template and importing pages, but something native and intuitive would be extremely beneficial.

**Comment by jpgreth** [comment]
Yeah, that's one of the features i would need the most

**Comment by matveyxsid** [comment]
+++ that would be great

**Comment by JVKdouk** [comment]
Looking forward to this one. Just duplicating an existing space would be enough for me, as I can build the template and then duplicate it

**Comment by FlavDumi** [comment]
Any plans to develop this feature?

**Comment by Philipinho** [comment]
Templates is available on the paid editions.
If you on the open-source edition, as a workaround, you could create space for templates and use the "Copy to space" feature for reuse.
